docs/modules/reference/pages/glossary.adoc

```diff
@@ -158,6 +175,14 @@ It also supports thresholding for streaming telemetry if you are using the xref:
 Snaps:: Self-contained software packages that run in a sandbox and have mediated access to host systems.
 Our Appliance Service uses snaps to distribute operating system packages and Minion service updates.
 
+Telemetry:: The process of automatically recording and transmitting data from an external source to a system in a different location for monitoring or analysis.
+
 Time series, time-series database (TSDB):: Time series is a sequence of data points that occur in successive order over a period of time.
 A time series database (TSDB) is designed to store and serve time series data.
-(See xref:deployment:time-series-storage/timeseries/time-series-storage.adoc[Time Series Storage] and https://en.wikipedia.org/wiki/Time_series_database[Time Series Database].)
\ No newline at end of file
+(See xref:deployment:time-series-storage/timeseries/time-series-storage.adoc[Time Series Storage] and https://en.wikipedia.org/wiki/Time_series_database[Time Series Database].)
+
+Topology:: A description of a network's elements (devices, services) and the connections between them.
+(See xref:operation:topology/introduction.adoc[Introduction to Topology].)
+
+Traps:: Event triggers collected from SNMP-capable devices on the network by the {page-component-title} Trapd service daemon.
```

Comment:
```suggestion
Traps:: Event triggers generated by SNMP-capable devices on the network and send to the {page-component-title} Trapd service daemon.
```

Reply:
Just fixing a minor tense error in the suggestion. :) 

```suggestion
Traps:: Event triggers generated by SNMP-capable devices on the network and sent to the {page-component-title} Trapd service daemon.
```